kodi.py

```diff
@@ -91,11 +91,14 @@
 def sanitize_name(media_name):
   # Normalize string
   normalized = unicodedata.normalize('NFKD', media_name).encode('ASCII', 'ignore')
-  # Strip symbols
-  m = RE_NAME_WITH_PARAM.match(normalized)
-  if m:
-    return m.group(1)
-  return media_name
+
+  # Strip things between and including brackets and parenthesis
+  removed_paren = re.sub(r'\([^)]*\)', '', normalized)
+  removed_bracket = re.sub(r'\[[^)]*\]', '', removed_paren)
```

Comment:
Might try with those again -- I think the fuzzy matching will catch those, at least "The Office".  Archer maybe not, but I'd be curious to see if it does.

An option might be a good idea, if we find it's necessary.  Just off the top of my head, I can think of another instance where you may want to keep what's inside the parentheses:

Some Song (feat One Band)
Some Song (feat Another Band)

There are probably other cases where the user would want to speak what's inside the parentheses.